app/controllers/api/v1/admin/roles_controller.rb

```diff
@@ -63,6 +63,9 @@ def update
         # Returns: { data: Array[serializable objects] , errors: Array[String] }
         # Does: Deletes a role.
         def destroy
+          default_roles = %w[User Administrator Guest].include?(@role.name)
```

Comment:
For verbosity and readibility,
It may be better to either call leave the logic as is but rename the `default_roles` to `is_default_role` since that what the variable will store and not the list of default roles,
Or, you keep its name but assign only the list to it and make the check on L67,

Reply:
Ahh I see what you mean